src/bindings/c/tests/ov_test.cpp

```diff
@@ -24,7 +24,7 @@ std::string plugins_xml_std = TestDataHelpers::generate_ieclass_xml_path("plugin
 #endif
 const char* plugins_xml = plugins_xml_std.c_str();
 
-std::map<ov_element_type_e, size_t> element_type_size_map = {{ov_element_type_e::BOOLEAN, 8},
+std::map<ov_element_type_e, size_t> element_type_size_map = {{ov_element_type_e::BOOLEAN, 1},
```

Comment:
Why did you change size to 1?
Boolean size if I remember right is not equal to `u1` and the size is 8. Or am I wrong?

Reply:
you are right, boolean is defined as 8bits data.

Reply:
I think will be better to have tests to validate that sizes of element type is the same with C++ data types.

Reply:
ok, will provide same element type test with C++ data type.